apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  API_SERVICES_CMS_BASE_PATH,
+  API_SERVICES_CMS_MOCKING,
+  API_SERVICES_CMS_URL
+} from "@/config/constants";
+import { Client, createClient } from "@/generated/services-cms/client";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import * as E from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import { NextRequest, NextResponse } from "next/server";
+
+if (API_SERVICES_CMS_MOCKING) {
```

Comment:
why do you want to setup mocks here in the module? can't we do this in the tests files? or in some setting at top level. generally, modules shouldn't have side effects when imported.

Reply:
This is done when you want to run the application locally and do not want to call external services APIs, all HTTP request to those external services will be intercepted by MSW and a mock response returned

Reply:
in any case removed